dependencies.yaml

```diff
@@ -0,0 +1,97 @@
+# Dependency list for https://github.com/rapidsai/dependency-file-generator
+files:
+  all:
+    output: conda
+    matrix:
+      cuda: ["11.5"]
+      arch: [x86_64]
+    includes:
+      - checks
+      - cudatoolkit
+      - py_version
+      - run
+      - test_notebooks
+      - test_python
+  test_notebooks:
+    output: none
+    includes:
+      - cudatoolkit
+      - py_version
+      - test_notebooks
+  test_python:
+    output: none
+    includes:
+      - cudatoolkit
+      - py_version
+      - test_python
+  checks:
+    output: none
+    includes:
+      - checks
+      - py_version
+channels:
+  - rapidsai
+  - rapidsai-nightly
+  - conda-forge
+  - nvidia
+dependencies:
+  cudatoolkit:
+    specific:
+      - output_types: conda
+        matrices:
+          - matrix:
+              cuda: "11.2"
+            packages:
+              - cudatoolkit=11.2
+          - matrix:
+              cuda: "11.4"
+            packages:
+              - cudatoolkit=11.4
+          - matrix:
+              cuda: "11.5"
+            packages:
+              - cudatoolkit=11.5
+  checks:
+    common:
+      - output_types: [conda, requirements]
+        packages:
+          - pre-commit
+  py_version:
+    specific:
+      - output_types: conda
+        matrices:
+          - matrix:
+              py: "3.8"
+            packages:
+              - python=3.8
+          - matrix:
+              py: "3.9"
+            packages:
+              - python=3.9
+          - matrix:
+            packages:
+              - python>=3.8,<3.10
+  run:
+    common:
+      - output_types: [conda, requirements]
+        packages:
+          - cupy >=10,<12.0.0a0
+          - numba >=0.49.0
+          - numpy >=1.19,<1.24
+          - scipy >=1.6.0
+  test_notebooks:
+    common:
+      - output_types: [conda, requirements]
+        packages:
+          - ipython
+          - matplotlib-base
+          - notebook
+          - pytorch <=1.12.1
```

Comment:
@ajschmidt8 We have been pinning pytorch as an x86_64 specific package, right? I know the notebook tests don’t run on ARM due to capacity but we might want to check and be consistent here. Especially with channels, if pytorch is preferred.

Reply:
good catch. resolved in d0cf417.